deploy/static/js/app.js

```diff
@@ -1,5 +1,25 @@
-$(function() {
-  $( '.js-focus' ).focus();
+// Adds a page load event that replaces "img.js-svg"-tags by an embedded version of the image.
+// As fetch()
+window.addEventListener(
+	"load",
+	() => {
+		let domParser = new DOMParser();
+		let embedSvgs = document.querySelectorAll("img.js-svg");
 
-  // Insert your application code here!
-});
+		for( let embedSvg of embedSvgs ) {
+			fetch(embedSvg["src"])
+				.then(response => response.text())
+				.then(svg => {
+					if( !svg )
+						return;
+
+					var svgElement = domParser.parseFromString(svg, "image/svg+xml");
+					svgElement = svgElement.querySelector("svg");
+
+					svgElement.classList = embedSvg.classList;
+					embedSvg.replaceWith(svgElement);
+				})
+				.catch();
```

Comment:
This catch looks pointless to me. Does this have a specific reason or did you forget an implementation?